struct-field-names-as-array/tests/tests.rs

```diff
@@ -26,6 +26,15 @@ struct TestSkip {
     #[field_names_as_slice(skip)]
     c: String,
 }
+#[derive(FieldNamesAsArray, FieldNamesAsSlice)]
+struct TestRename {
+    a: String,
+    b: String,
+    #[field_names_as_array(rename = "last_option")]
+    #[field_names_as_slice(rename = "last_option")]
+    c: String,
+}
```

Comment:
Hate to nitpick on such a nice PR, but I'd prefer spacing between items to be consistent:

```suggestion

#[derive(FieldNamesAsArray, FieldNamesAsSlice)]
struct TestRename {
    a: String,
    b: String,
    #[field_names_as_array(rename = "last_option")]
    #[field_names_as_slice(rename = "last_option")]
    c: String,
}
```